gcp/readme.md

```diff
@@ -0,0 +1,23 @@
+# GCP Deployment
+
+This Terraform module will deploy VMs on GCP Compute Engine.
+
+**Prerequisites:**
+- An existing subnet to deploy the VMs into. The subnet's attached firewall should allow inbound traffic on ports 22, 3000, 8888, 8889, 9090, 9092, 9644 and 33145. This module adds the `rp-node` tag to the deployed VMs, which can be used as the target tag for the firewall rule.
+
+After completing these steps, please follow the required steps in the [project readme](../README.md) to deploy Redpanda to the new VMs.
+
+1. `terraform init`
+2. `terraform apply` to create the resources on AWS.
```

Comment:
We are missing the `enable_monitoring` variable.

Reply:
Good catch. I had this ready and then I rebased and missed a couple updates.